crates/solver/src/main.rs

```diff
@@ -280,6 +281,19 @@ struct Arguments {
     /// but at the same time we don't restrict solutions sizes too much
     #[clap(long, env, default_value = "15000000")]
     simulation_gas_limit: u128,
+
+    /// In order to protect against malicious solvers, the driver will check that settlements prices do not
+    /// exceed a max price deviation compared to the external prices of the driver, if this optional value is set.
+    /// The max deviation value should be provided as a float percentage value. E.g. for a max price deviation
+    /// of 3%, one should set it to 0.03f64
+    #[clap(long, env)]
+    max_settlement_price_deviation: Option<f64>,
```

Comment:
nit: why not max_price_deviation?

Reply:
I prefer `max_settlement_price_deviation` here because for runtime arguments `max_price_deviation` would be super ambiguous (price deviation for what?).